homeassistant/components/august/lock.py

```diff
@@ -37,11 +37,16 @@ async def async_setup_entry(
     data: AugustData = hass.data[DOMAIN][config_entry.entry_id]
     async_add_entities(AugustLock(data, lock) for lock in data.locks)
 
+class LockEntityFeature(IntFlag):
+    """Supported features of the lock entity."""
+
+    OPEN = 1
```

Comment:
no need to redefine this, this can be imported `from homeassistant.components.lock`